### ISSUE TITLE
AE2's crafting/pattern terminal causing hard lock

### ISSUE THREAD

**Issue by DontCareIfHacked** [The-Nexus-Project/Limitless-3#250]
It's not a hard crash, more like a hard lock.

I'm not using OptiFine.
### Describe your issue, what happened?

I've been using AE2 for a long time. I've spent 10+ hours using the crafting and pattern terminal. I have 100+ patterns made, so at this point, I don't want to switch to refined storage. 
I've noticed every so often I experience a hard lock while using the crafting and pattern terminal. During this time my fps goes from 20-40fps to zero, the GPU activity (shown in task manager) goes from about 50% to less then 2%, and I the game stops taking any input, including escape. This isn't a hard crash, as I can still hear things happen in the background, such as cats meowing, people moving near me, and pikes hunting other fish.  If this happens to me while playing on a server, and the server admin kicks me, it will take me to the "you've been kicked...." screen, where minecraft is working properly again. At least, it does until I go to use a crafting or pattern terminal again, wherein I get the hard lock again. To be clear, as far as I can tell, the hard lock only happens when interacting with a crafting or pattern terminal - using chests and other things are fine. Once I get a hard lock, it doesn't matter if how many times I restart windows and minecraft, the hard lock continuously happens. I once even restarted my computer, then made a brand spanking new world in (creative) single player mode, loaded into the world without anyone else being able to join it, spent about three minutes making a very simple AE2 system, then right clicked on the crafting terminal, only to instantly experience the hard lock. So, in other words, once I experience one hard lock, it doesn't matter if I play on a server or offline in a single player world. I've found the only way to get around this is to delete two files: appliedenergistics2-client.toml and appliedenergistics2-common.toml . However, this is only temporarily, as the problem eventually reoccurs. 

Side note: I noticed when the hard lock occurs, it can cause server side lag for everyone else. It can also cause the server's mean tick time to increase by 20+ MS, while at the same time, the TPS can also drop. 

TLDR: I think there might be something wrong with mixin code and/or AE2's config file. I suspect an endless loop is being created. Or, "FPS reducer" somehow is being triggered into thinking you've suddenly gone afk, which probably is causing it to completely kill fps.

### To Reproduce

If you have more then 32gb (like me), make a private server running off of your computer. 
Then make sure all background processes (ie: chrome) aren't running, so you have 20+gb of free ram. Then log into the server using the same computer. Also make sure your client is using 8gb-10gb.
Once you're in, type in gibberish into JEI, so nothing shows up.
Then set up a complete AE2 system, including crafting towers, patterns, hard drives, power system, CPU cores, ect... Make sure as little as possible is showing up in JEI during this time.
If you haven't experienced your first hard lock yet, try making 400+ food using the crafting terminal. Chances are you'll experience it by the time you're done. If you still haven't, try leaving the crafting terminal open for 10+ continuous hours, then try closing the terminal. 
Once a hard lock occurs, you can try typing /kick <insert your minecraft name here> into the server console, to unbork your client. 

### Expected behavior
I expect to use AE2's crafting terminal and pattern terminal without experiencing a hard lock.

### Crash Reports & Logs (Don't paste entire crash reports here.)
It's a hard lock, not a hard crash. As such, it doesn't generate a crash log.

### Affected Version
Latest. :P Seriously though:

- Pack Version: Listed version shown through curse: 1.16.5hf(.zip). Pack version shown in game version shown (at main screen): 1.11.6
- On a server?: Yes, but this happens even in single player mode, where no one else except for me has access to the world.
